lib/foreman_tasks/engine.rb

```diff
@@ -69,6 +69,8 @@ class Engine < ::Rails::Engine
 
       config.to_prepare do
         ::Api::V2::HostsController.send :include, ForemanTasks::Concerns::HostsControllerExtension
+        ::PuppetclassesController.send :include, ForemanTasks::Concerns::EnvironmentsExtension
```

Comment:
Do we need to extend this controller as well? I don't see it defining the `obsolete_and_new`